gqueries/modules/etmoses/number_of_residences.gql

```diff
@@ -0,0 +1,2 @@
+- query = AREA(number_of_new_residences)
```

Comment:
Why `number_of_new_residences` and not `number_of_residences`? It would be a significant change in behaviour for ETMoses to only include new residences.